src/main/java/Die.java

```diff
@@ -0,0 +1,25 @@
+package main.java;
+
+import java.util.Random;
+
+public class Die {
+
+    private static int TOTAL_FACES = 6;
+
+    int faceValue = -1;
+
+    public Die() {}
+
+    public void roll() {
```

Comment:
Static methode may be?

Reply:
I wouldn't since the faceValue isn't static